__tests__/routes/profiles.test.js

```diff
@@ -48,4 +51,43 @@ describe('profiles router endpoints', () => {
       expect(res.body.error).toBe('ProfileNotFound');
     });
   });
+
+  describe('POST /profile', () => {
+    it('should return 200 when profile is created', async () => {
+      const profile = {
+        name: 'Louie Smith',
+        email: 'louie@example.com',
+        avatarUrl:
+          'https://s3.amazonaws.com/uifaces/faces/twitter/hermanobrother/128.jpg',
+      };
+      Profiles.findById.mockResolvedValue(undefined);
+      Profiles.create.mockResolvedValue([
+        Object.assign({ id: 'd376de0577681ca93614' }, profile),
+      ]);
+      const res = await request(server).post('/profile').send(profile);
```

Comment:
Maybe a comment here about what the mockResolvedValue is doing?
It took me a few minutes to tease out how this is working.